partiql-ast/src/main/java/org/partiql/ast/DataType.java

```diff
@@ -38,7 +38,7 @@ public static class StructField extends AstNode {
 
         private final boolean optional;
 
-        @Nullable
+        @NotNull
```

Comment:
(self-review) lists should be `@NotNull` rather than `@Nullable`. Taken from https://github.com/partiql/partiql-lang-kotlin/pull/1692